CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## (IN PROGRESS)
 
+* "Based on" fields required for encumbrance rollover even when not in use. Refs UIF-274
```

Comment:
```suggestion
* "Based on" fields not required for encumbrance rollover when not in use. Refs UIF-274
```

Do we articulate the bug or the fix here?

Reply:
usually I copy-paste title of ticket